LongLookupTables/requirements.txt

```diff
@@ -0,0 +1,5 @@
+numpy
```

Comment:
In general we would like to keep the number of dependencies low. The less dependencies, the more likely that people will use our library (and it is also less work to maintain). Here it is fine, but in machine we don't want to use pandas, sklearn, scipy (actually even numpy, although it is now listed as a dependency). Do you think these libraries are essential here?

Reply:
I completely agree with you about dependencies. I decided to go with pandas because it made the whole code a lot easier and more readable (then it needs numpy). Normally I'm reticent to use such dependencies (for example I would never add pandas or sklearn to `machine`) but I thought that for a single script generator it's fine. I could remove scipy and sklearn but then I would either have to remove the `stratify` parameter or add quite a bit of code for that.

Reply:
ok let's leave them here!